packages/php-wasm/logger/src/lib/logger.ts

```diff
@@ -248,14 +292,40 @@ export function collectPhpLogs(
 }
 
 /**
- * Add a listener for the fatal Playground errors.
- * These errors include Playground errors like Asyncify errors. PHP errors won't trigger this event.
+ * Collect worker metrics.
+ *
+ * @param worker The service worker
+ */
+export function collectWorkerMetrics(worker: ServiceWorkerGlobalScope) {
+	worker.addEventListener('activate', (event) => {
+		event.waitUntil(worker.clients.claim());
+	});
+	worker.addEventListener('message', (event) => {
+		if (event.data === 'getClientInfo') {
+			worker.clients.matchAll().then((clients) => {
+				if (!event.source) {
```

Comment:
this check could be done before `if (event.data ===`

Reply:
Done [c37a47e](https://github.com/WordPress/wordpress-playground/pull/1253/commits/c37a47e593b8203df5212ccb109e95120a7df895)